### PR TITLE
Fix #212: Make the hiding controls optional

### LINKED ISSUE #212: Make the hiding of the controls optional
User submitted feature request.

### PR DESCRIPTION
**Pull request by aayushsingla**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Run the unit tests with `gradlew app:connectedGmsDebugAndroidTest` to make sure you didn't break anything (with an emulator running or phone connected)

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/java/com/google/android/stardroid/activities/util/FullscreenControlsManager.java [hunk 1]
```diff
@@ -87,18 +89,21 @@ public void delayHideTheControls() {
...
       return false;
     }
   };
 
   private void toggle() {
-    if (mVisible) {
-      hide();
-    } else {
-      show();
+    if (autoHideEnabled()) {
```
Comment (jaydeetay) [inline app/src/main/java/com/google/android/stardroid/activities/util/FullscreenControlsManager.java:96]: Great. Can you remove the AUTO_HIDE constant too if it's no longer used?

Comment (aayushsingla) [reply app/src/main/java/com/google/android/stardroid/activities/util/FullscreenControlsManager.java:96]: yeah sure. Will update within a day

Comment (jaydeetay) [reply app/src/main/java/com/google/android/stardroid/activities/util/FullscreenControlsManager.java:96]: Thanks! I don't mind doing that myself in a follow up, but it seemed to me it belongs in this change.  Appreciate the contribution.

**Comment by aayushsingla** [comment]
@jaydeetay done the changes. Actually, I should have taken care of this in my first commit itself.

**Comment by jaydeetay** [comment]
Cheers!

**Pull request merged**